packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.v1.vue

```diff
@@ -83,6 +83,16 @@
                 :is="errorBoundary"
                 tier="3">
                 <slot name="meta-items" />
+
+                <delivery-time-meta
+                    v-if="deliveryTimeData.eta || deliveryTimeData.distance || deliveryTimeData.address"
+                    data-test-id="restaurant-delivery-time-meta"
+                    :address="deliveryTimeData.address"
+                    :distance="deliveryTimeData.distance"
+                    :distance-for-screen-readers="deliveryTimeData.distanceForScreenReaders"
+                    :eta="deliveryTimeData.eta"
+                    :eta-for-screen-readers="deliveryTimeData.etaForScreenReaders"
+                    :is-list-item="isListItem" />
```

Comment:
```suggestion
                <delivery-time-meta
                    v-if="deliveryTimeData.eta || deliveryTimeData.distance || deliveryTimeData.address"
                    v-bind="deliveryTimeData"
                    data-test-id="restaurant-delivery-time-meta"
                    :is-list-item="isListItem" />
```

Reply:
love it!!!! completely forgot to do it. thanks!